### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by ajtruckle** [johngodley/redirection#1641]
```
Plugin: 4.2
WordPress: 5.1.1 (single)
PHP: 7.2.17
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36
JavaScript: https://www.publictalksoftware.co.uk/wp-content/plugins/redirection/redirection.js
REST API: https://www.publictalksoftware.co.uk/wp-json/

Error: Failed to fetch

Error: Failed to fetch
```

**Comment by ajtruckle** [comment]
![Error3](https://user-images.githubusercontent.com/7985331/55685166-f5ee5500-594a-11e9-9181-e474e3810b1b.jpg)

**Comment by johngodley** [comment]
The problem is shown at the top - something on your site is likely blocking the REST API.

**Comment by ajtruckle** [comment]
I don't know what though? I have not changed my site.

**Comment by ajtruckle** [comment]
I have recently upgraded plugins like WP Security Audit Log.

**Comment by ajtruckle** [comment]
As we found out, exporting to JSON works but not exporting to CSV.

**Comment by johngodley** [comment]
A typo prevented the CSV download from working. Fixed in 4.2.1
